dotcom-rendering/src/components/Branding.importable.tsx

```diff
@@ -78,6 +78,10 @@ function decideLogo(
 	 *  so fallback to standard logo if not present */
 	const maybeDarkLogo = branding.logoForDarkBackground ?? branding.logo;
 
+	const isMedia =
+		format.design === ArticleDesign.Video ||
+		format.design === ArticleDesign.Audio;
```

Comment:
Technically it's also ArticleDesign.Gallery and ArticleDesign.Picture which also have dark backgrounds